dev-python/ipython/ipython-7.14.0.ebuild

```diff
@@ -91,6 +91,7 @@ src_test() {
 
 python_test() {
 	distutils_install_for_testing
+	sed -i 's/^def \(test_embed_svg_url():$\)/def _\1/g' build/test/lib/IPython/core/tests/test_display.py || die
```

Comment:
Just edit the source file in `prepare_all`, also please wrap lines.

Reply:
Fixed